### PR TITLE
Add PeptideDB

### PR DESCRIPTION
**Pull request by ZimmerA**
PeptideDB

### CODE REVIEW

File: recipes/peptidedb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% set version = "0.0.1" %}
+{% set version_string = "0.0.1-beta" %}
```
Comment (bgruening) [inline recipes/peptidedb/meta.yaml:2]: if the version of the package is 0.0.1 but the version_string is 0.0.1-beta ... what happens if you are out of beta? Will this be 0.0.2? We usually recommend to only package stable software. Is that possible here?

File: recipes/peptidedb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - dotnet tool install --add-source $SRC_DIR --tool-path $PREFIX/bin --version {{ version_string }} PeptideDB
+
+requirements:
+  build:
```
Comment (bgruening) [inline recipes/peptidedb/meta.yaml:20]: can this also be `host` and not specify anything under build?

File: recipes/peptidedb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    - dotnet-sdk
+  run:
+    - dotnet-sdk
+
+test:
+  commands:
+    - dotnet tool list --tool-path $PREFIX/bin
```
Comment (bgruening) [inline recipes/peptidedb/meta.yaml:27]: is it also possible to call something like `PeptideDB --help` here?

File: recipes/peptidedb/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+about:
+  home: https://csb.bio.uni-kl.de/
+  license: MIT
+  summary: PeptideDB
```
Comment (bgruening) [inline recipes/peptidedb/meta.yaml:32]: A somehow more meaningful help would be nice. And a description with some more text.
There is also something like `dev_url:` where you can point to the github repo if you like.

**Review by bgruening — COMMENTED** [review]
Thanks @ZimmerA! Some comments inline.

**Comment by ZimmerA** [comment]
> Thanks @ZimmerA! Some comments inline.

Thank you for the tips @bgruening!
I addressed them in my latest commits and we should be good to merge now.

**Comment by ZimmerA** [comment]
@bioconda/core this should be ready to merge

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@ZimmerA cool beans!

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**